g11n-ws/vip-manager-i18n/src/main/resources/application-s3.properties

```diff
@@ -0,0 +1,80 @@
+##
+#Copyright 2019 VMware, Inc.
+#SPDX-License-Identifier: EPL-2.0
+##
+
+# SPRING CONFIG (ConfigFileApplicationListener)
+spring.config.name= # config file name (default to 'application')
+spring.config.location= # location of config file
+
+# IDENTITY (ContextIdApplicationContextInitializer)
+spring.application.name=Singleton
+
+# logging
+logging.config=classpath:log4j2-spring.xml
+
+# EMBEDDED SERVER CONFIGURATION (ServerProperties)
+server.port=8090
+server.scheme=https
+server.https.key-store=classpath:vip.jks
+server.https.key-store-password=vmware2018
+server.https.key-password=vmware2018
+server.https.key-store-type=JKS
+server.https.key-alias=server
+server.http.port=8091
+server.trace.enable=false
+
+#collection source server
+source.cache.flag = false
+source.cache.server.url = https://localhost:8088
+
+#S3 store config
+s3.accessKey=###########
+s3.secretkey=##########
+s3.region=############
```

Comment:
please put the real values to these keys, they are not sensitive:
s3.region=############
s3.bucketName=##########